src/windowsdesktop/src/sfx/Microsoft.WindowsDesktop.App.Runtime.sfxproj

```diff
@@ -64,7 +64,7 @@
     <FrameworkListFileClass Include="System.Windows.Forms.Primitives.resources.dll" Profile="WindowsForms" />
     <FrameworkListFileClass Include="System.Windows.Forms.resources.dll" Profile="WindowsForms" />
     <FrameworkListFileClass Include="System.Windows.Input.Manipulations.resources.dll" Profile="WindowsForms" />
-    <FrameworkListFileClass Include="System.Private.Windows.Core.dll"/>
+    <FrameworkListFileClass Include="System.Private.Windows.Core.dll" Profile="WindowsForms" />
```

Comment:
Did you confirm that setting this value will work in a publish setting? Perhaps by manually adding this to the RuntimeList.xml on an older runtime pack?

Reply:
Yes, I had tried this and see that System.Private.Windows.Core.dll is now included in publish folder when publishing self contained winforms app